frontend/src/slicer/productSlice.ts

```diff
@@ -50,13 +56,13 @@ export const getOneProduct = createAsyncThunk<IResponseGetAllProducts, string, {
 export const createProduct = createAsyncThunk<IResponseGetAllProducts, void, { state: RootState, dispatch: Dispatch }>(
     'products/create',
     async (_, {getState, dispatch}) => {
-        if (!validateProduct(getState())) {
+        if (!validateBeforeSendingToBackend(getState())) {
```

Comment:
`getState` is very generic, which data exactly do you get here?

Reply:
getState is one of reduxjs toolkit's Dispatch Object's keys. It gets the RootState of the redux store.
 I guess I could do something like {getState: getRootState ...} while destructuring one line above but I think I'll stick to the naming of @reduxjs/toolkit here